.github/workflows/ci.yml

```diff
@@ -52,7 +52,7 @@ jobs:
     strategy:  
       matrix: 
         include:
-          - os: ubuntu-16.04
+          - os: ubuntu-20.04
```

Comment:
Just highlighting this change, as it is not clear if this is desired. I just moved this up, because with `ubuntu-16-04` the CI pipeline in my personal repo fails because there is no Ubuntu 16.04 available anymore. You may want to change this back.